### PR TITLE
docs: document new filters UI PEM-6202

### PR DESCRIPTION
**Pull request by addetz**
## Describe the Change

This PR refactors and documents cluster filters and the map view.

## Changed Pages

💻 [Map and Filter Clusters](https://deploy-preview-4611--docs-spectrocloud.netlify.app/clusters/cluster-management/cluster-map-filters/)

## Jira Tickets

🎫 [PEM-6202](https://spectrocloud.atlassian.net/browse/PEM-6202)

## Backports

Can this PR be backported?

- [x] No. _Please leave a short comment below about why this PR cannot be backported._
Merging to a release.

[PEM-6202]: https://spectrocloud.atlassian.net/browse/PEM-6202?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/clusters/cluster-management/cluster-map-filters.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+title: "Map and Filter Clusters"
+description: "Learn how to view your cluster locations and filter your clusters."
+hide_table_of_contents: false
+sidebar_position: 180
+tags: ["clusters", "cluster management"]
+---
+
+The Palette Dashboard provides users with the ability to map and filter Kubernetes clusters. You can find these
+capabilities on the **Clusters** page. Mapping and filtering is available for clusters deployed to public clouds, data
+centers and edge hosts.
```
Comment (benradstone) [inline docs/docs-content/clusters/cluster-management/cluster-map-filters.md:12]:
```suggestion
centers, and edge hosts.
```

File: docs/docs-content/clusters/cluster-management/cluster-map-filters.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+### Prerequisites
+
+- A [Palette](https://console.spectrocloud.com) account with the permissions to view and list clusters. Refer to the
+  [Roles and Permissions](../../user-management/palette-rbac/project-scope-roles-permissions.md) guide for more
+  information.
+- One or more active Kubernetes clusters in Palette.
+
+### Enablement
+
+1. Login to [Palette](https://console.spectrocloud.com).
```
Comment (benradstone) [inline docs/docs-content/clusters/cluster-management/cluster-map-filters.md:30]:
```suggestion
1. Log in to [Palette](https://console.spectrocloud.com).
```

File: docs/docs-content/clusters/cluster-management/cluster-map-filters.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+   | **Name**        | Filter and display the cluster matching the specified name.                      | Custom string                                                                            |
+   | **Environment** | Filter and display clusters based on the cluster type.                           | Configured cluster types                                                                 |
+   | **Status**      | Filter and display clusters based on their current status.                       | **Pending**, **Provisioning**, **Running**, **Deleting**, **Importing**, and **Unknown** |
+   | **Profiles**    | Filter and display clusters based on the name of the configured cluster profile. | Custom string                                                                            |
+
+   ![Cluster list view](/clusters_cluster-management_cluster-map-filters_cluster-list-view.webp)
+
+3. Click on the **Environment** drop-down list and select a cluster type. The cluster list displays matching clusters.
```
Comment (benradstone) [inline docs/docs-content/clusters/cluster-management/cluster-map-filters.md:45]:
```suggestion
3. Click on the **Environment** drop-down Menu and select a cluster type. The cluster list displays matching clusters.
```

File: docs/docs-content/clusters/cluster-management/cluster-map-filters.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+4. Click on the **Status** drop-down list and select the **Running** status. The cluster list displays clusters matching
```
Comment (benradstone) [inline docs/docs-content/clusters/cluster-management/cluster-map-filters.md:47]:
```suggestion
4. Click on the **Status** drop-down Menu and select the **Running** status. The cluster list displays clusters matching
```

File: docs/docs-content/clusters/cluster-management/cluster-map-filters.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+   | **Status**          | Filter and display clusters based on their current status.                                                                                                                                              | **Pending**, **Provisioning**, **Running**, **Deleting**, **Importing**, and **Unknown** |
+   | **Profiles**        | Filter and display clusters based on the name of the configured cluster profile.                                                                                                                        | Custom string                                                                            |
+   | **Tags**            | Filter and display clusters which have the specified tag. Tags can be specified during cluster deployment or from the **Cluster Settings** tab.                                                         | Custom string                                                                            |
```
Comment (benradstone) [inline docs/docs-content/clusters/cluster-management/cluster-map-filters.md:77]:
```suggestion
   | **Tags**            | Filter and display clusters with the specified tag. Tags can be specified during cluster deployment or from the **Cluster Settings** tab.                                                         | Custom string                                                                            |
```

File: docs/docs-content/clusters/cluster-management/cluster-map-filters.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+
+   <summary>Configure Cluster Location</summary>
+
+   Select the cluster to you want to update.
+
+   Click on **Settings**. Then, select **Cluster Settings**. The **Settings** pane appears.
+
+   Select the **Location** option. Search for the cluster location and click on **Save Changes**.
+
+   Repeat these steps for all the clusters that are missing a location.
```
Comment (benradstone) [inline docs/docs-content/clusters/cluster-management/cluster-map-filters.md:121]: Could these be changed to numbered steps?

Just wondering if there was a reason if they weren't.

File: docs/docs-content/clusters/cluster-management/cluster-map-filters.md [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+   | **Name**        | Filter and display the cluster matching the specified name.                      | Custom string                                                                            |
+   | **Environment** | Filter and display clusters based on the cluster type.                           | Configured cluster types                                                                 |
+   | **Status**      | Filter and display clusters based on their current status.                       | **Pending**, **Provisioning**, **Running**, **Deleting**, **Importing**, and **Unknown** |
+   | **Profiles**    | Filter and display clusters based on the name of the configured cluster profile. | Custom string                                                                            |
+
+   ![Cluster map view](/clusters_cluster-management_cluster-map-filters_cluster-map-view.webp)
+
+4. Click on the **Environment** drop-down list and select a cluster type. The map displays matching clusters.
```
Comment (benradstone) [inline docs/docs-content/clusters/cluster-management/cluster-map-filters.md:138]:
```suggestion
4. Click on the **Environment** drop-down Menu and select a cluster type. The map displays matching clusters.
```

File: docs/docs-content/clusters/cluster-management/cluster-map-filters.md [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+
+5. Click on the **Status** drop-down list and select **Running** status. The map displays clusters matching the selected
```
Comment (benradstone) [inline docs/docs-content/clusters/cluster-management/cluster-map-filters.md:140]:
```suggestion
5. Click on the **Status** drop-down Menu and select **Running** status. The map displays clusters matching the selected
```

**Review by benradstone — CHANGES_REQUESTED** [review]
Just a few changes requested to match our style guide.

Everything else looks great.

File: docs/docs-content/clusters/cluster-management/cluster-map-filters.md (hunk 6 above)
Comment (addetz) [reply docs/docs-content/clusters/cluster-management/cluster-map-filters.md]: Added! Thanks for the suggestion!

**Review by karl-cardenas-coding — APPROVED** [review]
Great changes

File: docs/docs-content/clusters/cluster-management/cluster-map-filters.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (addetz) [reply docs/docs-content/clusters/cluster-management/cluster-map-filters.md]: Added his change manually since the suggestion is Outdated.

**Review by benradstone — APPROVED** [review]

**Pull request merged**